src/projectile.ts

```diff
@@ -1,32 +1,28 @@
 import { GameField } from "./gamefield";
 
+export type DirectionsType = 'up' | 'down' | 'left' | 'right';
```

Comment:
I don't quite understand what the export type means.

Reply:
type is for defining the values which can variable has, export just alows to use this type in other files